cmd/destroy/destroy.go

```diff
@@ -178,12 +188,23 @@ func Destroy(ctx context.Context) *cobra.Command {
 			}
 			os.Setenv("KUBECONFIG", kubeconfigPath)
 			defer os.Remove(kubeconfigPath)
-			err = c.runDestroy(ctx, options)
-			analytics.TrackDestroy(err == nil)
-			if err == nil {
-				oktetoLog.Success("Development environment '%s' successfully destroyed", options.Name)
-			}
 
+			// when option --all the cmd will destroy everything at the namespace and return
+			if options.DestroyAll {
+				if !okteto.Context().IsOkteto {
+					return errors.New("option `--all` is not available for non-Okteto clusters. Learn more: https://www.okteto.com/docs/self-hosted/")
+				}
+				err = c.runDestroyAll(ctx, options)
```

Comment:
Should we track the destroy here?

Reply:
It is being tracked in line 207, right? We are adding a new option/flag `DestroyAll`

Reply:
I though it was in the else branch 😅